apps/hash-api/src/index.ts

```diff
@@ -223,6 +242,39 @@ const main = async () => {
   const authMiddleware = createAuthMiddleware({ logger, context });
   app.use(authMiddleware);
 
+  /**
+   * Add scope to Sentry, now the user has been checked.
+   * We could set some of this scope earlier, but it doesn't get picked up for GraphQL requests for some reason
+   * if the middleware comes earlier.
+   */
+  app.use((req, _res, next) => {
+    const scope = Sentry.getCurrentScope();
+
+    // Clear the scope and breadcrumbs – requests seem to bleed into each other otherwise
+    scope.clear();
+    scope.clearBreadcrumbs();
+
+    /**
+     * Sentry automatically populates a 'Headers' object, but for some reason it doesn't do this for GraphQL requests.
+     * This might be something to do with how Sentry hooks into fetch that doesn't play nicely with ApolloServer,
+     * or how we're loading it.
+     */
```

Comment:
Does it help if you add `new Sentry.Integrations.GraphQL()` to the `initSentry` function?

Reply:
Just tried it, didn't seem to do anything